runtime/stdlib/contracts/test.cdc

```diff
@@ -2,160 +2,154 @@
 ///
 pub contract Test {
 
-    /// Blockchain emulates a real network.
+    /// backend emulates a real network.
     ///
-    pub struct Blockchain {
+    pub let backend: AnyStruct{BlockchainBackend}
 
-        pub let backend: AnyStruct{BlockchainBackend}
+    init(backend: AnyStruct{BlockchainBackend}) {
+        self.backend = backend
+    }
 
-        init(backend: AnyStruct{BlockchainBackend}) {
-            self.backend = backend
-        }
+    /// Executes a script and returns the script return value and the status.
+    /// `returnValue` field of the result will be `nil` if the script failed.
+    ///
+    pub fun executeScript(_ script: String, _ arguments: [AnyStruct]): ScriptResult {
```

Comment:
While at it, maybe also replace `pub` with `access(all)` for all these methods, so won't run into conflicts when we sync this with stable-cadence branch

Reply:
Updated in https://github.com/onflow/cadence/pull/2783/commits/72bbfe1e5d4a5ed438762cfe05a6c7b1c1f731ef